src/R/Support/Impl/Help/Packages/PackageInfo.cs

```diff
@@ -80,7 +80,7 @@ internal sealed class PackageInfo : NamedItemInfo, IPackageInfo {
                 try {
                     var result = await _host.Session.PackagesFunctionsNamesAsync(Name, REvaluationKind.BaseEnv);
                     functions = result.Children<JValue>().Select(v => (string)v.Value);
-                } catch (TaskCanceledException) { } catch (REvaluationException) { }
+                } catch (RException) { }
```

Comment:
Yes, I just mean that take `LoadWorkspaceCommand` - it handles both

```
        private async Task LoadWorkspace(IRSession session, string file) {
            try {
                await session.LoadWorkspaceAsync(file);
            } catch (RException ex) {
                var message = string.Format(CultureInfo.CurrentCulture, Resources.LoadWorkspaceFailedMessageFormat, file, ex.Message);
                VsAppShell.Current.ShowErrorMessage(message);
            } catch (OperationCanceledException) {
            }
        }
```

but in `WorkingDirectoryCommand` `SetDirectory` does not, only handles `RException`